x/oracle/types/asset.go

```diff
@@ -12,3 +17,26 @@ const (
 	BlocksPerYear   = BlocksPerDay * 365
 	MicroUnit       = int64(1e6)
 )
+
+type (
+	// GenesisState - all oracle state that must be provided at genesis
+
+	// ExchangeRatePrevote - struct to store a validator's prevote on the rate of Luna in the denom asset
```

Comment:
nit: can we cleanup the godocs to be like `ExchangeRatePrevote defines a structure to ...`. This includes all the types here 👍